### ISSUE TITLE
Loop Feature or play again

### ISSUE THREAD

**Issue by rickyazharipratama** [florent37/Flutter-AssetsAudioPlayer#16]
hi..., is there any chance to make sound loop ? 

Also i have no idea audioPlayer.finished.listen never called even the music is already end. here is the log  of the end of music.
```
V/MediaPlayer-JNI(18285): getCurrentPosition: 138001 (msec)
V/MediaPlayer-JNI(18285): getCurrentPosition: 138304 (msec)
V/MediaPlayer-JNI(18285): getCurrentPosition: 138607 (msec)
V/MediaPlayerNative(18285): message received msg=2, ext1=0, ext2=0
V/MediaPlayerNative(18285): playback complete
V/MediaPlayerNative(18285): callback application
V/MediaPlayerNative(18285): back from callback
V/MediaPlayerNative(18285): message received msg=211, ext1=0, ext2=0
V/MediaPlayerNative(18285): unrecognized message: (211, 0, 0)
V/MediaPlayerNative(18285): callback application
V/MediaPlayerNative(18285): back from callback
V/MediaPlayer-JNI(18285): getCurrentPosition: 138767 (msec)
I/chatty  (18285): uid=10277(com.teamcoret.findpairs.findpairs) identical 1 line
V/MediaPlayer-JNI(18285): getCurrentPosition: 138767 (msec)
V/MediaPlayer-JNI(18285): getCurrentPosition: 138767 (msec)
I/chatty  (18285): uid=10277(com.teamcoret.findpairs.findpairs) identical 1 line
V/MediaPlayer-JNI(18285): getCurrentPosition: 138767 (msec)
V/MediaPlayer-JNI(18285): getCurrentPosition: 138767 (msec)
```

and the log currentPosition always loop.

**Comment by hokamc** [comment]
You may change `ValueObservable<bool> get finished => _isPlaying.stream;` to `ValueObservable<bool> get finished => _finished.stream;` first.
To be careful, finished return null only when called.

**Comment by florent37** [comment]
thanks @hokamc I fixed this, and added a Loop Feature in 1.2.2
